src/utils/svelte.ts

```diff
@@ -0,0 +1,22 @@
+import { isQuoted, isWrappedWith } from './common';
+
+/**
+ * Indicates whether the attribute value is a Svelte interpolation.
+ *
+ * ---
+ *
+ * Example interpolation:
+ * ```
+ * a(href="{ cat.id }")
+ * ```
+ *
+ * In this case `val` is `"{ cat.id }"`.
+ *
+ * ---
+ *
+ * @param val Value of tag attribute.
+ * @returns `true` if `val` passes the svelte interpolation check, otherwise `false`.
+ */
+export function isSvelteInterpolation(val: string): boolean {
+	return val.length >= 3 && isQuoted(val) && isWrappedWith(val, '{', '}', 1) && !val.includes('{', 2);
```

Comment:
The expression `isWrappedWith(val, '{', '}', 1) && !val.includes('{', 2)` seems to be repeated similarly in the angular.ts.

How about extracting it into in another method?

````js
function isWrappedWithExclusive(val: string, start: string, end: string, offset: number) {
    return isWrappedWith(val, start, end, offset) && !val.includes(start, start.length + offset);
}
````

Reply:
Theoretically a good idea, but IMO we start to over-engineer this :smile:  
Let's let it as is for now and if there are more cases in the future, I will rethink about this extraction
But until now, it's already a one-liner :sweat_smile: